### PR TITLE
Add ALU joke

### PR DESCRIPTION
**Pull request by TheMatthew**
They are bad

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -889,3 +889,11 @@ Lisp programmers don't make prank calls. They make `FUNCALL`s
 **A**: In a dad-a-base.
 
 ---
+
+**Q**: Why did the ALU refuse to add two subnibbles.
+
+**A**: Because it refuses to be seen doing a two bit operation.
```
Comment (styfle) [inline readme.md:895]:
```suggestion
**A**: Because it refuses to be seen doing a two-bit operation.
```

File: readme.md [hunk 2]
```diff
@@ -889,3 +889,11 @@ Lisp programmers don't make prank calls. They make `FUNCALL`s
...
+
+---
+
+I have proof I should not be treated object. Obviously, I have no class.
```
Comment (styfle) [inline readme.md:899]:
```suggestion

```

This is similar to the other class jokes, lets remove it

File: readme.md (hunk 1 above)
Comment (MatthewKhouzam) [reply readme.md:895]: Thanks!

File: readme.md (hunk 2 above)
Comment (MatthewKhouzam) [reply readme.md:899]: I missed them, thanks for the sanity check!

**Review by styfle — APPROVED** [review]
🤓

**Pull request merged**